build.gradle

```diff
@@ -5002,10 +5011,10 @@ compileTargets { t ->
             }
         }
 
-        def modularPublicationJarName = "${moduleName}-${t.name}.jar"
         def modularPublicationJarTask = project.task("modularPublicationJar${t.capital}", type: Jar, dependsOn: modularEmptyPublicationJarTask) {
             destinationDirectory = file("${dstModularJarDir}")
-            archiveFileName = modularPublicationJarName
+            archiveBaseName = moduleName
+            archiveClassifier = t.name
```

Comment:
Same question here.

Reply:
Pulling this up from the `publications {}` block (where this information is removed) into the "Model of the Software".